examples/02_decoding/plot_miyawaki_encoding.py

```diff
@@ -0,0 +1,233 @@
+"""
+============================================================
+Encoding models for visual stimuli from Miyawaki et al. 2008
+============================================================
+
+This example demonstrates how to build an encoding model for
+functional MRI data with visual stimuli.
+
+Encoding models try to predict neuronal activity using properties of
+a real world stimulus, like images or sound.
+
+In the dataset from :func:`nilearn.datasets.fetch_miyawaki2008`,
+participants were shown images consisting of 10x10
+binary (either black or white) pixels and the corresponding  fMRI activity
+was recorded. We will predict the neuronal activity from the pixel-values
+of the presented images for each voxel. Then we extract the receptive fields
+for a set of voxels to see which pixel location a voxel is most sensitive to.
+
+See also :doc:`plot_miyawaki_reconstruction` for a decoding
+approach for the same dataset.
+"""
+
+##############################################################################
+# Loading the data
+# ----------------
+# First some imports:
+
+import numpy as np
+import pylab as plt
+from nilearn.plotting import plot_stat_map
+from nilearn import masking
```

Comment:
In examples, we like to do the imports in the same "cell" as where we use the corresponding symbols: it makes it easier for beginner to relate the symbols to where they come from.